src/Tracy/assets/Bar/bar.css

```diff
@@ -190,6 +190,8 @@ body #tracy-debug {
 	color: #575753;
 	margin: -5px -5px 5px;
 	padding: 0 25px 5px 5px;
+	max-width: 700px;
+	overflow: hidden;
```

Comment:
Mayby `word-wrap: break-word;` will be better than overflow hidden, can you test it in your use case?